Doc/library/inspect.rst

```diff
@@ -817,6 +817,19 @@ function.
        .. versionchanged:: 3.10
          The *globals*, *locals*, and *eval_str* parameters were added.
 
+   .. classmethod:: Signature.from_frame(frame)
+
+       Return a :class:`Signature` (or its subclass) object for a given
+       :ref:`frame object <frame-objects>`.
+
+       Notice that it is impossible to get signatures
+       with annotations from frames.
+       Because annotations are stored in functions.
+       Also note that default parameter values might
+       be modified inside the frame.
```

Comment:
This sentence is misleading. Default parameter values are not overridden. It's more than the function populate arguments from local variables, and it's displayed as default values. I'm not sure how to explain that.